modules/acquia_cms_article/acquia_cms_article.install

```diff
@@ -50,3 +51,17 @@ function acquia_cms_article_update_8002() {
     $field->save();
   }
 }
+
+/**
+ * Implements hook_update_N().
+ *
+ * Update the Pathauto pattern configuration for deprecated 'Node' bundle.
+ */
+function acquia_cms_article_update_8003() {
+  $module_path = \Drupal::moduleHandler()->getModule('acquia_cms_article')->getPath();
+  $config_optional = $module_path . '/config/optional';
+  $source_optional_dir = new FileStorage($config_optional);
+
+  $config_storage = \Drupal::service('config.storage');
+  $config_storage->write('pathauto.pattern.article', $source_optional_dir->read('pathauto.pattern.article'));
```

Comment:
@panshulK We can't change the URL alias using by importing the config file because it might happen user would have changed the URL alias and this will overwrite their changes. Please take a look at update hook: `pathauto_update_8108()`. Please change it across all update hooks.

Reply:
Done @vishalkhode1.

Reply:
Thanks. Changes looks good now.